.github/workflows/main.yml

```diff
@@ -114,6 +114,56 @@ jobs:
           # would be no failed golden test to upload.
           if-no-files-found: ignore   
   
+  ios-integration-test:
+    runs-on: macos-latest
+    timeout-minutes: 45
+    strategy:
+      matrix:
+        device:
+          - "iPhone 13"
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '11'
```

Comment:
For what is that required? Why the specific distribution?    
If its copied from an example somewhere then lets put the link in a comment above it. Otherwise in the future we might not know on the spot why its there and it might cause confusion.

Reply:
This can be removed 👍 It's only needed when using Flutter + Android. But for iOS we don't need this